core/templates/domain/exploration/WrittenTranslationObjectFactory.ts

```diff
@@ -23,19 +23,15 @@ import { Injectable } from '@angular/core';
 export const WRITTEN_TRANSLATION_TYPE_HTML = 'html';
 export const WRITTEN_TRANSLATION_TYPE_UNICODE = 'unicode';
 
-export type WrittenTranslationDataFormat = (
-  typeof WRITTEN_TRANSLATION_TYPE_UNICODE |
-  typeof WRITTEN_TRANSLATION_TYPE_HTML);
-
 export interface TranslationBackendDict {
-  'data_format': WrittenTranslationDataFormat;
+  'data_format': string;
   'translation': string;
   'needs_update': boolean;
 }
 
 export class WrittenTranslation {
   constructor(
-      public dataFormat: WrittenTranslationDataFormat,
+      public dataFormat: string,
```

Comment:
In the backend it looks like you accept several types here. How come this has changed to string?